### PR TITLE
Bump pulse-publisher

### PR DESCRIPTION
**Pull request by jonasfj**
Please feel free to land this... Should fix the pulse issue we had tuesday...

### CODE REVIEW

**Comment by djmitche** [comment]
I'll let you handle the landing, particularly given the schema validation error in Travis

File: schemas/authenticate-hawk-request.yml [hunk 1]
```diff
@@ -43,9 +43,9 @@ properties:
       **Note,** order of querystring elements is important.
   host:
     type:                 string
```
Comment (djmitche) [inline schemas/authenticate-hawk-request.yml]: does this need to be removed?

Comment (jonasfj) [inline schemas/authenticate-hawk-request.yml]: Sorry, I got it wrong... I remove the other `type: string`, I don't know if this works... But why wouldn't it, I suspect it does :)

**Comment by jonasfj** [comment]
I'll land when travis passes...

**Pull request merged**